Workbooks/HDInsight/At-Scale/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"HDInsight At-Scale Insights",
```

Comment:
Nit: you can probably remove "At-scale" from the title of this template, so it just displays as "HDInsight Insights" ?  whatever text is here will be displayed in the gallery.  you should also add a description here.